pkgs/development/python-modules/rich/default.nix

```diff
@@ -1,40 +1,46 @@
 { lib
 , buildPythonPackage
-, fetchFromGitHub
-, pythonOlder
-, CommonMark
 , colorama
+, CommonMark
 , dataclasses
+, fetchFromGitHub
 , ipywidgets
-, poetry
+, poetry-core
 , pygments
-, typing-extensions
 , pytestCheckHook
+, pythonOlder
+, typing-extensions
 }:
```

Comment:
Some people sort these packages by the order they are used in, so I wouldn't apply sorting here. Also makes diffs harder to read.